src/com/google/javascript/jscomp/ProductionCoverageInstrumentationCallback.java

```diff
@@ -250,10 +228,14 @@ private Node newInstrumentationNode(Node node, String fileName, String fnName, T
       columnNo = node.getParent().getCharno();
     }
 
-    Node innerProp = IR.getprop(IR.name(MODULE_RENAMING), IR.string(INSTRUMENT_CODE_INSTANCE));
-    Node outerProp = IR.getprop(innerProp, IR.string(INSTRUMENT_CODE_FUNCTION_NAME));
+    String encodedParam = parameterMapping.getEncodedParam(fileName, fnName, type, lineNo, columnNo);
+
+
+   // Node innerProp = IR.getprop(IR.name(MODULE_RENAMING), IR.string(INSTRUMENT_CODE_INSTANCE));
+   // Node outerProp = IR.getprop(innerProp, IR.string(INSTRUMENT_CODE_FUNCTION_NAME));
+    Node prop = IR.getprop(IR.name(INSTRUMENTATION_ARRAY), IR.string("push"));
```

Comment:
The JavaDoc for this function should also be updated.

Unfortunately I can't comment directly on the JavaDoc :( https://github.com/isaacs/github/issues/1655.